releases/mainnet/1.9.11/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.9.11</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.11</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.11/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmQHSHnaRPSJrnLihtTyMPBTHxZHTcBTDLfQBhvsAK7pAn">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.9.11/abi/index.html">ABIs</a></li>
+
+				<li>Links to GitHub repositories
+					<ul>
+						<li>DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.9.11" target="_blank">changelog</a>)</li>
```

Comment:
I'll get this created soon, so I think is fine to keep it. Otherwise it can be commented and I'll send a new PR to uncomment it.